### DISCUSSION TITLE
[Bug] Cannot find path 'C:\Users\xxxxx\scoop\shims\shims' because it does not exist.

### DISCUSSION THREAD

**Discussion by ahmetkca** [ScoopInstaller/Scoop#5402] [Help and Questions]
#### Current Behavior
Cannot use the `scoop` with any subcommands!
Getting the following error when I run `scoop`:
```
Get-ChildItem: C:\Users\ahmet\scoop\apps\scoop\current\lib\commands.ps1:2
Line |
   2 |  … tem "$PSScriptRoot\..\libexec") + (Get-ChildItem "$scoopdir\shims") |
     |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\ahmet\scoop\shims\shims' because it does not exist.
```

#### Expected Behavior
it shouldn't throw the above command and properly execute the given command.

#### Additional context/output
```
Get-ChildItem: C:\Users\ahmet\scoop\apps\scoop\current\lib\commands.ps1:2
Line |
   2 |  … tem "$PSScriptRoot\..\libexec") + (Get-ChildItem "$scoopdir\shims") |
     |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\xxxxx\scoop\shims\shims' because it does not exist.
```

```
> echo $env:scoop
C:\Users\ahmet\scoop\shims
```

### System details

**Windows version: 11

**OS architecture: 64bit

**PowerShell version:
```
Major  Minor  Patch  PreReleaseLabel BuildLabel
-----  -----  -----  --------------- ----------
7      3      2
```

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
  "last_update": "2023-01-28T00:42:17.0898902-05:00",
  "scoop_repo": "https://github.com/ScoopInstaller/Scoop",
  "scoop_branch": "master"
}
```

**Comment by rashil2000** [comment]
How did you install Scoop? Why is your SCOOP variable set to the shims folder?

**Reply by ahmetkca** [reply]
I don't exactly remember. It was a while ago. However, I think I installed with `iwr -useb get.scoop.sh | iex`.

**Comment by ahmetkca** [comment]
I honestly have no idea why it all of sudden stopped working. I renamed the `scoop` folder to `backup-scoop` and reinstalled it again. Completely closed the Windows Terminal and reopened it again. Now it is working. So, if this happens to anybody the only solution is just make a backup of your `scoop` folder and reinstall again, I guess.
